src/endpoint.zig

```diff
@@ -68,7 +68,9 @@ fn buildRoutes(EP: anytype) [routeCount(EP)]Router.Match {
 
 const std = @import("std");
 const Allocator = std.mem.Allocator;
-const Verse = @import("verse.zig");
+const router = @import("router.zig");
+const server = @import("server.zig");
```

Comment:
```suggestion
const Server = @import("server.zig");
```
it's a type with fields